components/FourPicsTable.js

```diff
@@ -1,38 +1,42 @@
-import Image from "next/image.js";
-import styled from "styled-components";
+import { StyledImage } from "./StyledImage";
 import { StyledA } from "./StyledLink";
+import styled from "styled-components";
 
-export default function ChooseFourPics() {
+export default function FourPicsTable() {
   return (
     <FourPicsContainer>
-      <StyledA href="https://i.imgur.com/SglfvOg.jpeg/">
-        <FourPics
+      <StyledA href="/PreviewPage/">
```

Comment:
I don't get what this `a` element is supposed to do. Right now, no matter which image I choose I always see the same picture.
Don't you have to pass an `imageID` to the `href` attribute?

Reply:
you are absolutely right but i want to implement the dynamic routing when my database is up and running. these PR are just for the "frontend-side" of things. 
thanks for your comments, you are exactly describing what i will do later on :)